services/galaxy-toolshed/galaxy-toolshed-activity.tester.js

```diff
@@ -0,0 +1,19 @@
+import { isFormattedDate } from '../test-validators.js'
+import { ServiceTester } from '../tester.js'
+
+export const t = new ServiceTester({
+  id: 'galaxy-toolshed',
+  title: 'GalaxyToolshed',
+})
```

Comment:
Please use the helper functions for the boilerplate items, more details in our [guide on writing service tests](https://github.com/badges/shields/blob/master/doc/service-tests.md#1-boilerplate)

Reply:
I tried but the character "-" in the service name raise an error. So, it's inspired from `f-froid` service

Reply:
F-Droid is a bad point of comparison because the hyphen is actually part of the service name (the service name is literally "F-Droid"), not a hyphen we introduce into the file name convention.

That's not the case here as I understand it; Galaxy and Toolshed are separate, non-hyphenated words. As such we should not have a `-` in the class name, which in turn won't cause any issues with the service registration. If you follow the standard prescriptions for naming the file and the service class (e.g. `GalaxyToolshedDownloads`) things should just work